site/docs/.vitepress/shared/syntaxes/env.tmLanguage.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
How did you obtain this definition?

Reply:
https://github.com/mikestead/vscode-dotenv/blob/ad506a66ede7d6215cb1f2a16c169eadd414916e/syntaxes/env.YAML-tmLanguage

Reply:
Oh alright